dashboard-for-web/articles/web-dashboard-designer-mode/ui-elements/dialogs-and-wizards/query-builder.md

```diff
@@ -1,54 +1,54 @@
 ---
 title: Query Builder
-author: Natalia Kazakova
+author: Margarita Zakhodyaeva
 legacyId: 117967
 ---
 # Query Builder
-In the **Query Builder** dialog, you can add data tables and views to the data source, and select which columns to include. The **Query Builder** automatically joins the related tables, so all you need to do is drag-and-drop.
+In the **Query Builder** dialog, you can add data tables and views to the data source, and select which columns to include. The **Query Builder** automatically joins the related tables, so all you need to do is drag and drop.
 
-![wdd-query-builder](../../../images/img124934.png)
+![wdd-query-builder](../../../../images/img124934.png)
 
-This topic consists of the following sections.
+This topic consists of the following sections:
 - [Add Tables](#add-tables)
 - [Join Tables](#join-tables)
 - [Edit Column Settings](#edit-column-settings)
 - [Filter Data](#filter-data)
 - [Preview Data](#preview-data)
 
 ## Add Tables
-To add the required tables/views to a data source, drag-and-drop it from the **Tables** pane onto the **Diagram** pane.
+To add the required tables/views to a data source, drag-and-drop them from the **Tables** pane onto the **Diagram** pane.
```

Comment:
Thanks for the clarification. Since this is the first sentence in the "Add Tables" section, I'd like to switch the sentence order:

```suggestion
Drag the required tables/views from the **Tables** pane onto the **Diagram** pane to add them to a data source.
```